contracts/contracts/interfaces/IPoolDeposits.sol

```diff
@@ -5,6 +5,8 @@ contract IPoolDeposits {
 
   function usersDeposit(address userAddress) external view returns (uint256);
 
+   function changeProposalAmount(uint256 amount) external;
```

Comment:
I'll run the linter on all of these. It is also on the todo list to refine our linting.